opentelemetry-sdk/src/opentelemetry/sdk/_logs/__init__.py

```diff
@@ -319,9 +321,27 @@ def __init__(
 
     @staticmethod
     def _get_attributes(record: logging.LogRecord) -> Attributes:
-        return {
+        attributes = {
             k: v for k, v in vars(record).items() if k not in _RESERVED_ATTRS
         }
+        if record.exc_info is not None:
+            exc_type = ""
+            message = ""
+            stack_trace = ""
+            exctype, value, tb = record.exc_info
+            if exctype is not None:
+                exc_type = exctype.__name__
+            if value is not None and value.args:
+                message = value.args[0]
+            if tb is not None:
+                # https://github.com/open-telemetry/opentelemetry-specification/blob/9fa7c656b26647b27e485a6af7e38dc716eba98a/specification/trace/semantic_conventions/exceptions.md#stacktrace-representation
+                stack_trace = "".join(
+                    traceback.format_exception(*record.exc_info)
```

Comment:
Discussed in SIG this should be good for now and good to keep it consistent with tracing